requirements.txt

```diff
@@ -0,0 +1,4 @@
+discord.py~=1.5.1
+requests~=2.25.0
```

Comment:
```suggestion
discord.py~=1.7.3
requests~=2.25.1
```

Reply:
requests~=2.25.0 and requests~=2.25.1 are equivalent, the ~= operator will use the latest compatible version (2.25.x)